mayland-blocks/block-templates/archive.html

```diff
@@ -3,7 +3,7 @@
 <!-- wp:group {"tagName":"main","layout":{"inherit":true}} -->
 <main class="wp-block-group">
 
-<!-- wp:query {"query":{"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":""}} -->
+<!-- wp:query {"query":{"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":"exclude","inherit":true}} -->
```

Comment:
Do we want to exclude sticky posts?

Reply:
Excluded because it's archive page. Do all pages show sticky posts?
Also I see that some themes such as `skatepark` exclude and some themes doesn't. Will have to check all themes on this.